Source/Scene/BillboardCollection.js

```diff
@@ -84,7 +84,7 @@ define([
         positionLowAndRotation : 1,
         compressedAttribute0 : 2,        // pixel offset, translate, horizontal origin, vertical origin, show, texture coordinates, direction
         compressedAttribute1 : 3,        // aligned axis, translucency by distance, image width
-        compressedAttribute2 : 4,        // image height, color, pick color, 2 bytes free
+        compressedAttribute2 : 4,        // image height, color, pick color, 15 bits free
```

Comment:
@lilleyse we will likely have a small merge conflict when we merge your instancing branch into master since I expect this will be merged first.